### DISCUSSION TITLE
部品の注文と必要な機材

### DISCUSSION THREAD

**Discussion by shoko93** [algyan/XIAOGYAN#60] [Ideas]
必要な機材のほとんどは秋月で購入できるので必要であれば部品と一緒に注文するようにドキュメントや[イベント](https://algyan.connpass.com/event/286572/)で補足の説明があるといいと思う。

**Comment by matsujirushi** [comment]
アドバイスありがとうございます！

[イベント](https://algyan.connpass.com/event/286572/)で話題にする予定です。（なので、主なメニューに「機材を確認しておこう」が入っていますw）

どの秋月製品を買うのが良いでしょうか？ ＞みなさん

**参考:**

* [機材の確認](https://github.com/algyan/XIAOGYAN/blob/main/manuals/assembly/README.md#14-%E6%A9%9F%E6%9D%90%E3%81%AE%E7%A2%BA%E8%AA%8D)

**Reply by shoko93** [reply]
よかったです。順番的に部品の発注→機材の確認になってたので、部品注文してしまったあとに気づくということにならないか心配してました。（秋月の場合注文確定すると変更できないので）

個人的に使っているのはこの辺りです
- [はんだごて](https://akizukidenshi.com/catalog/g/gT-02535/)
- [こて台](https://akizukidenshi.com/catalog/g/gT-06870/)
- [はんだ](https://akizukidenshi.com/catalog/g/gT-02594/)
- [ピンセット](https://akizukidenshi.com/catalog/g/gT-02541/)
- [テスター](https://akizukidenshi.com/catalog/g/gM-01288/)
- [はんだ吸い取り器](https://akizukidenshi.com/catalog/g/gT-03639/)

テスターはもう少し安いのでも十分かもしれません。

**Reply by matsujirushi** [reply]
@tatsu1225 

* 組立マニュアルに秋月電子通商の通販コードを掲載
* 部品手配マニュアルからリンク参照

という形にするのが良いと思うのですが、どうでしょう？

**Reply by tatsu1225** [reply]
秋月で買える機材を眺めてきました．約5000円で済む最小構成と約10000円のちょっといいヤツの2パターンでまとめてみました．が，結構悩ましいですね．
使ったことが無い機材だと，自信をもって推しづらい...特にニッパーとテスター．
[秋月縛りの機材選定.xlsx](https://github.com/algyan/XIAOGYAN/files/11724776/default.xlsx)

**Reply by tatsu1225** [reply]
@matsujirushi はい、通販コードが定まったらそんな形にするのが良いと思います！

**Reply by matsujirushi** [reply]
道具への投資の考え方（とりあえず安くとか、長く使いたいとか）がひとそれぞれだと思いますし、販売状況が変わったときのドキュメント更新が大変なので、、、
あくまで参考ということで組立マニュアルとは別に**道具のドキュメント**を用意しようと思います。

ご協力いただける詳しい方が出てきたら改訂してもらうことにしましょう。

**Reply by tatsu1225** [reply]
機材候補を眺めていて思いましたが、まさしくそれで、どのような考え方で勧めるかは難しいと感じていましたので、参考としての道具ドキュメントに賛成です。

**Reply by matsujirushi** [reply]
**機材マニュアル**を追加しました。
